auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/RestWrapper.java

```diff
@@ -83,26 +83,22 @@ public void setRecord(Object record) {
 
     @JsonProperty("TotalRecordCount")
     public int getTotalRecordCount() {
-        if (records != null) {
-            if (records instanceof Collection) {
+
+            if ( records != null && records instanceof Collection) {
                 Collection c = (Collection) records;
                 for (Object obj : c) {
                     try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)